Assets/MirageXR/Tests/NewUI/ProfileView.cs

```diff
@@ -295,4 +295,10 @@ private void UpdateConectedLRS(DBManager.LearningRecordStores publicCurrentLearn
                 break;
         }
     }
+
+    private void CloseAndReturnToHomePage()
+    {
+        Close();
+        RootView_v2.Instance.ReturnToHomePage();
+    }
```

Comment:
why do we have to go back to the home screen instead of just closing the profile window?

Reply:
to switch the toggle on the bottom panel

Reply:
@cawke happy with the answer?

Reply:
fixed